src/components/material/material-buttons/online/MaterialButtonOnlineExternal.tsx

```diff
@@ -44,10 +47,27 @@ const MaterialButtonOnlineExternal: FC<MaterialButtonOnlineExternalProps> = ({
     }
   }, [data, error, translatedUrl, urlWasTranslated]);
 
+  const label = (sourceName: string, materialTypes: string[]) => {
```

Comment:
Is there a type from the generated graphql.tsx file we can use for material types?

Reply:
Yes indeed, good sir! I am adjusting it now :)